enterprise/ha/src/main/java/org/neo4j/kernel/ha/lock/SlaveLocksClient.java

```diff
@@ -240,6 +241,12 @@ public int getLockSessionId()
         return initialized ? client.getLockSessionId() : -1;
     }
 
+    @Override
+    public Collection<Locks.ActiveLock> activeLocks()
+    {
+        throw new UnsupportedOperationException( "not implemented" );
```

Comment:
This one should also be able to use the `sharedLocks` and `exclusiveLocks` maps.